README.md

```diff
@@ -1870,6 +1870,26 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='doc-comments-before-attributes'></a>(<a href='#doc-comments-before-declarations'>link</a>) **Place doc comments for a declaration before any attributes.** [![SwiftFormat: docCommentsBeforeAttributes](https://img.shields.io/badge/SwiftFormat-docCommentsBeforeAttributes-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#docCommentsBeforeAttributes)
+
+  <details>
+
+  ```swift
+  // WRONG
+
+  @MainActor
+  /// A spacecraft with everything you need to explore the universe.
+  struct Spaceship { … }
+
+  // RIGHT
+
+  /// A spacecraft with everything you need to explore the universe.
+  @MainActor
```

Comment:
Does this apply to property wrappers and builtin attributes like `@objc` or `@dynamicMemberLookup` as well?

Reply:
Yes! It applies to everything that starts with an `@`. 

(In the Swift grammar everything that starts with an `@` is an attribute -- you can confirm this by searching for `@` in the [grammar documentation](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/summaryofthegrammar) and seeing that it only appears once, as the start of an `attribute`.)